Pinta.Gui.Widgets/Dialogs/SimpleEffectDialog.cs

```diff
@@ -348,96 +380,33 @@ private static Gtk.Label CreateHintLabel (string hint)
 		return label;
 	}
 
+	private void SetAndNotify (MemberReflector settings, object o, object val)
```

Comment:
`settings` should probably be named something like `reflector` here instead?

Reply:
Sure. I just renamed it